src/functionalTest/java/uk/gov/hmcts/reform/pbis/SubscriptionProcessingTest.java

```diff
@@ -0,0 +1,137 @@
+package uk.gov.hmcts.reform.pbis;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static org.assertj.core.api.Java6Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import org.awaitility.Duration;
+import org.junit.Before;
+import org.junit.Test;
+import uk.gov.hmcts.reform.pbis.model.PrivateBetaRegistration;
+import uk.gov.hmcts.reform.pbis.utils.ServiceBusFeeder;
+import uk.gov.service.notify.Notification;
+import uk.gov.service.notify.NotificationClient;
+import uk.gov.service.notify.NotificationClientException;
+
+
+public class SubscriptionProcessingTest {
+
+    private static final Configuration config = new Configuration();
+
+    private static final Duration MAX_WAIT_TIME_MS = new Duration(
```

Comment:
no need for unit of time in variable imho as you've used Duration type

Reply:
good point, will change that